File: achieve/code_test/programmers/문자열내p와y의개수.py
def solution(s):
    answer = True

    text = s.lower()
    p = 0
    y = 0

    for i in text:
        if i == "p":
            p += 1
        if i == "y":
            y += 1

    if (p == y):
        answer = True
    else:
        answer = False

    return answer

File: achieve/code_test/programmers/문자열내p와y의개수.js
function solution(s) {
  var answer = true;
  let text = s.toLowerCase();
  let key = 0;

  for (let i of text) {
    if (i == "p") {
      key++;
    }
    if (i == "y") {
      key--;
    }
  }

  if (key == 0) {
    answer = true;
  } else {
    answer = false;
  }

  return answer;
}

function solution2(s) {
  var answer = true;
  let text = s.toLowerCase();
  let p = 0;
  let y = 0;

  for (let i of text) {
    if (i == "p") {
      p++;
    }
    if (i == "y") {
      y++;
    }
  }

  if (p == y) {
    answer = true;
  } else {
    answer = false;
  }

  return answer;
}
